### DISCUSSION TITLE
What name would you like to give your child? あなたのこどもにつけたいと思う名前を教えて下さい。

### DISCUSSION THREAD

**Discussion by n-takuma** [NAIST-SE/AttractingContributors#435] [General]
あなたのこどもにつけたいと思う名前を教えて下さい。男の子、女の子問わないです。

What name would you like to give your child?　It does not matter if it is a boy or a girl.

My fork: 
[README.md](https://github.com/NAIST-SE/AttractingContributors/files/8891590/README.md)
https://github.com/n-takuma/AttractingContributors-ChiledName

**Comment by aru1702** [comment]
Currently I don't know, maybe something with "Light" ?
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
